orchestrator-restapi/src/main/java/com/yahoo/vespa/orchestrator/restapi/wire/GetHostResponse.java

```diff
@@ -2,27 +2,48 @@
 package com.yahoo.vespa.orchestrator.restapi.wire;
 
 import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.annotation.JsonProperty;
 
+import java.util.List;
 import java.util.Objects;
 
 /*
  * @author andreer
  */
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
 public class GetHostResponse {
 
     public static final String FIELD_NAME_HOSTNAME = "hostname";
     public static final String FIELD_NAME_STATE = "state";
+    public static final String FIELD_NAME_APPLICATION_URL = "applicationUrl";
+    public static final String FIELD_NAME_SERVICES = "services";
 
     private final String hostname;
     private final String state;
+    private final String applicationUrl;
+    private final List<HostService> services;
+
+    // Deprecated - kept for backwards compatibility until clients have migrated away
+    public GetHostResponse(String hostname, String state) {
```

Comment:
Consider adding a deprecation annotation

Reply:
Aren't warnings treated as errors? I plan to remove this method later today, once I have updated the other repos.

Reply:
Yes, but you can annotate all methods using the deprecated feature with SupressWarning("deprecation").

Reply:
OK, good to know.